source/renderer/app/components/status/formatCpuInfo.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import type { Cpu } from '../../types/systemInfoTypes';
+import { formattedNumber } from '../../utils/formatters';
+
+const formatCpuInfo = ({ model, speed }: Cpu): string => {
+  const modelName = model.split('@')[0].trim().replace(/\s+/g, ' ');
+  const speedValue = formattedNumber(speed / 1000, 2);
+
+  return `${modelName} @ ${speedValue}GHz`;
```

Comment:
Do we need to take into account cases where either model name or speed is not going to be available? To avoid something like: `MyUnknownProcessor @ 0GHz`?

Reply:
Probably such case would be impossible, however I agree. Let's be more defensive. I will consider three cases:
1. No model property - return empty string
2. No speed - return only formatted model name.
3. return full information